src/style.css

```diff
@@ -68,7 +68,7 @@ form {
 input,
 select,
 textarea {
-  color: #000;
+  color: rgb(0, 0, 0);
 }
 
```

Comment:
-This first item must be of index 1 and then you scale... So five should be index 5 not 4 https://github.com/Besufekad-HAZ/To-do-list-testing/blob/602d0e17fce5ee84fd9db5202e6cbeb7cf267c9b/src/tasks-list.js#L17-L25